src/Opaleye/Constant.hs

```diff
@@ -132,6 +134,24 @@ instance (D.Default Constant a (Column b), T.IsSqlType b)
          => D.Default Constant [a] (Column (T.PGArray b)) where
   def = Constant (T.pgArray (constantExplicit D.def))
 
+instance D.Default Constant (R.PGRange Int.Int) (Column (T.PGRange T.PGInt4)) where
+  def = Constant $ \(R.PGRange a b) -> T.pgRange T.pgInt4 a b
+
+instance D.Default Constant (R.PGRange Int.Int64) (Column (T.PGRange T.PGInt8)) where
+  def = Constant $ \(R.PGRange a b) -> T.pgRange T.pgInt8 a b
+
+-- TODO
+--instance D.Default Constant (R.PGRange _) (Column (T.PGRange PGNumeric)) where
```

Comment:
I'm not a fan of commented code. Unless you're planning on adding the missing instance in this PR could you remove it?